ci/avalon-direct-model-sgx-tests.yaml

```diff
@@ -71,9 +71,10 @@ services:
       - 5555
     command: |
       bash -c "
-        avalon_listener --bind http://avalon-listener:1947 --lmdb_url http://avalon-lmdb:9090
+        avalon_listener --bind http://avalon-listener:1947 --lmdb_url http://avalon-lmdb:9090 --zmq_url tcp://avalon-enclave-manager:5555
```

Comment:
In docker mode, can we read the zmq url from config file . Default zmq url can be tcp://avalon-enclave-manager:5555.   Only in standalone mode we can pass the zmq_url as command line arguent. For worker pool, we need to use polling mode. So passing zmq url in docker mode is not required.

Reply:
Either is fine. I am following consistent approach where other services are following like listener and lmdb.

Reply:
Zmq url is needed only in sync mode and in case of docker we can read default value from config file. Moreover for worker pool we will use polling approach. So let's not add zmq url in docker-compose file IMO.

Reply:
We are not disabled sync mode yet. IMO we shoudn't hard code container service name in config file, same thing is passed as argument in the command. It will helps in QA or others to not modify the config file otherwise we have to maintain multiple config files.